src/components/search-panel/search-panel.css

```diff
@@ -0,0 +1,47 @@
+.nested-container {
+    margin-left: 1rem
+}
+
+.container {
+    margin-left: 1.5rem
+}
+
+.record-type {
+    height: 10px;
```

Comment:
Let's follow the attribute ordering.
1. float, positioning (if needed)
2. display (if flex add below flex rules)
3. layout (width, height, margin, padding, border - in that order)
3. font rules
4. others